stacks/openshift/application/oidc-authservice-for-appid/README.md

```diff
@@ -2,18 +2,31 @@
 
 ## Prerequisites
 
-* Provisioning an AppID instance from IBM Cloud. the free tier works.
-* keep the configuration parameters including:
-    - client_id
-    - OIDC provider URL
-    - client_secret
+* Provisioning an [AppID](https://cloud.ibm.com/catalog/services/app-id) 
+instance from IBM Cloud. It can start with the _Lite_ plan, but will need the
+_Graduated tier_ once you need more than 1000 authentication events per month.
+* Create an application with type _reguarwebapp_ under the provioned AppID
+instance. Make sure the caope contains `email` and retrieve the following
+configuration parameters from your AppID. They will be used to configure the
+OIDC auth service:
+    - `clientId`
+    - `secret`
+    - `oAuthServerUrl`
 
 ## How to use
 
-After provisioning the statefulset `authservice` in namespace `istio-system`,
-update the environment variables via either web console or CLI with parameters
-from AppID instance.
+this configuration should be included as part of kubeflow deployment. After a successful Kubeflow deployment, you will find no pod running in the statefulset
```

Comment:
this -> This

Reply:
Updated in https://github.com/IBM/manifests/pull/6/commits/71726186ee3483f09db7fc0130c73642e55c751d